packages/GenderSelection/src/GenderSelection.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState, AllHTMLAttributes, FC, RefObject, RefAttributes, useRef, useEffect} from 'react';
+import classnames from 'classnames';
+import styles from './GenderSelection.scss';
+import { Input } from "@mbkit/input";
+import { Select } from "@mbkit/select";
+import { Label } from "@mbkit/label";
+
+export type SelectOptions = {
```

Comment:
You should be able to import and export the types from `@mbkit/select` so they aren't duplicated